app/src/main/res/values/strings.xml

```diff
@@ -63,111 +63,111 @@
-    <string name="new_badge_earnt_message">Hooray! You have just earnt a new badge.</string>
+    <string name="new_badge_earnt_message">Hooray! You have just earned a new badge.</string>
     <!-- Message dialog -->
     <string name="btnOk">Ok</string>
     <string name="messageDialogErrorTitle">Error</string>
-    <string name="messageDialogGenericError">I\'m so sorry. Something didn\'t work.\nPlease, try again or contact the developer.</string>
-    <string name="messageDialogErrorCode">Something didn\'t work. Please, try again or contact the developer.\nError code: EX-{{*{{error_code}}*}}</string>
+    <string name="messageDialogGenericError">That didn\'t work.\nPlease try again, or contact the developer.</string>
+    <string name="messageDialogErrorCode">That didn\'t work. Please try again, or contact the developer.\nError code: EX-{{*{{error_code}}*}}</string>
     <string name="youHaveToAcceptPrivacyPolicyTitle">Login failed</string>
-    <string name="youHaveToAcceptPrivacyPolicy">You need to do the login on Common Voice website and you have to accept Privacy Policy.</string>
+    <string name="youHaveToAcceptPrivacyPolicy">You must first log in on the Common Voice website and accept the Common Voice privacy policy.</string>
     <string name="youHaveToAcceptPrivacyPolicyOpenCommonVoice">Open Common Voice</string>
     <!-- Daily goal -->
     <string name="daily_goal_title">Daily goal</string>
-    <string name="daily_goal_is_not_set">Daily goal is not set yet</string>
-    <string name="set_daily_goal">Set goal now</string>
+    <string name="daily_goal_is_not_set">No goal set</string>
+    <string name="set_daily_goal">Set a goal</string>
     <string name="edit_daily_goal">Edit goal</string>
     <string name="save_daily_goal">Save</string>
     <string name="delete_daily_goal">Delete</string>
     <string name="cancel_daily_goal">Cancel</string>
-    <string name="description_setting_daily_goal">Choose how many sentences you want to record and clips you want to validate (sum of them) per day</string>
+    <string name="description_setting_daily_goal">Choose how many sentences you want to record and clips you want to validate (sum of them) per day.</string>
     <string name="daily_goal_achieved_today">Achieved today</string>
     <string name="daily_goal_not_achieved_today">Not achieved today</string>
-    <string name="daily_goal_achieved_message">Well done! You have just achieved the daily goal.\nExactly, you validated {{*{{n_clips}}*}} and recorded {{*{{n_sentences}}*}}.\nAnyway you can continue to contribute to Common Voice.</string>
-    <string name="daily_goal_is_not_official_by_mozilla">This daily goal isn\'t official by Mozilla. It\'s only a feature of this app.</string>
+    <string name="daily_goal_achieved_message">Well done! You achieved your daily goal.\nYou validated {{*{{n_clips}}*}} and recorded {{*{{n_sentences}}*}}.\nThank you for contributing to Common Voice!</string>
```

Comment:
Here is different the meaning of the sentence... my idea was to inform user that he/she can continue to contribute, even if he/she achieved the daily goal.

Reply:
I see. I've edited this sentence again for you.